assets/_sass/_navbar.scss

```diff
@@ -24,11 +24,10 @@ $nav-item-border: darken($navbar-default-bg, 8%);
 }
 
 #resizer {
-  display: none;
+  display: block;
```

Comment:
The resizer should be display: none by default and only show when you hover over it